README.md

```diff
@@ -6,12 +6,12 @@ This is a [Viam module](https://docs.viam.com/manage/configuration/#modules) for
 
 ### Checking Python version
 
-First and foremost, open a terminal on your robot, and run the following commands to check its Python and pip versions:
+First and foremost, open a terminal on your robot, and run the following commands to check its root Python and pip versions:
 
 ```console
-python3 --version
-python3 -m ensurepip --default-pip
-pip3 --version
+sudo python3 --version
+sudo python3 -m ensurepip --default-pip
+sudo pip3 --version
```

Comment:
We also need to check for `python3-venv` here right.

Ubuntu and debian have python3 (3.8.x) preinstalled. So I guess we really just need to check for `python3-pip` and `python3-venv`.

Reply:
Might include check for python3 itself still in case someone wants to test on a macbook